### PR TITLE
Add translation support for the AltaPay Payment Actions grid

### PR DESCRIPTION
**Pull request by maf-embraceit**
... in the following WordPress-supported languages `Danish, German, Estonian, Finnish, French, Czech, German (Austria), German (Switzerland, informal), German (formal), German (Switzerland), French (Belgium), French (Canada), Italian, Lithuanian, Dutch, Dutch (Belgium), Dutch (Formal), Norwegian Nynorsk, Polish, Romanian, Swedish.`

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes reviewed.

File: languages/altapay-da_DK.po [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+msgid "Payment Released."
+msgstr "Betaling frigivet."
+
+#: altapay.php:385
+msgid "Order got no transaction"
+msgstr "Ordren har ingen transaktion"
+
+#: altapay.php:708
+msgid "Payment Captured."
+msgstr "Betaling indfanget."
```
Comment (lbmdk) [inline languages/altapay-da_DK.po:48]: Betaling hentet

File: languages/altapay-da_DK.po [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+#: altapay.php:832
+msgid "Payment refund is in progress."
+msgstr "Betalingsrefusion er i gang."
```
Comment (lbmdk) [inline languages/altapay-da_DK.po:52]: Refundering for betaling er i gang

File: languages/altapay-da_DK.po [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+#: views/tables/index.blade.php:66
+#: views/tables/index.blade.php:101
+msgid "Amount"
+msgstr "Beløb"
+
+#: views/tables/index.blade.php:68
+#: views/tables/index.blade.php:139
+#: views/tables/index.blade.php:143
+msgid "Capture"
+msgstr "Indfang"
```
Comment (lbmdk) [inline languages/altapay-da_DK.po:107]: Hent
Indhent

File: languages/altapay-da_DK.po [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+#: views/tables/index.blade.php:123
+#: views/tables/index.blade.php:124
+msgid "Chargeable"
+msgstr "Debiterbar"
+
+#: views/tables/index.blade.php:128
+#: views/tables/index.blade.php:130
+msgid "Captured"
+msgstr "Indfanget"
```
Comment (lbmdk) [inline languages/altapay-da_DK.po:131]: Hentet
Indhentet

File: languages/altapay-da_DK.po [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+#: views/tables/index.blade.php:133
+#: views/tables/index.blade.php:136
+msgid "Refunded"
+msgstr "Refunderet"
+
+#: views/tables/index.blade.php:138
+#: views/tables/index.blade.php:142
+msgid "Capture Payment"
+msgstr "Indfang betaling"
```
Comment (lbmdk) [inline languages/altapay-da_DK.po:141]: Hent
Indhent

File: languages/altapay-pl_PL.po [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2024-07-18T10:05:00+00:00\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"X-Generator: WP-CLI 2.10.0\n"
+"X-Domain: altapay\n"
+
+
+#: altapay.php:254
+msgid "AltaPay Payment Actions"
+msgstr "Działania płatnicze AltaPay"
```
Comment (mzasonski) [inline languages/altapay-pl_PL.po:21]: Not sure of the context here, but this sounds artificial.

If it's title to the section where we list possible actions to be done for given payment then something like that will be better:
```
Operacje na płatności AltaPay
```

File: languages/altapay-pl_PL.po [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+#: views/tables/index.blade.php:103
+#: views/tables/index.blade.php:144
+#: views/tables/index.blade.php:148
+msgid "Refund"
+msgstr "Zwrot"
+
+#: views/tables/index.blade.php:113
+msgid "Transaction ID"
+msgstr "ID transakcji"
```
Comment (mzasonski) [inline languages/altapay-pl_PL.po:117]: Possibly to use full polish translation (`ID` abbreviation is not commonly used in Polish apart from people knowing English):

```
Identyfikator transakcji
```

**Review by emicha — APPROVED** [review]

**Pull request merged**